### DISCUSSION TITLE
add postResumeCommand?

### DISCUSSION THREAD

**Discussion by acrollet** [community/community#3527] [Codespaces] [answered]
Hi there,

it doesn't look like a hook for running commands after resuming (similar to `postCreateCommand`) exists. Is one planned? Thanks for any feedback!

**Comment by srivatsn** [comment, marked as answer]
There is a `postStartCommand` that will run for all container starts (i.e on create and resume) - will that work for your scenario or do you need something that only runs on resume? /cc @lostintangent

**Reply by acrollet** [reply]
Ah, that should do it, I'll try it tomorrow. Thanks!

**Reply by mchelen-gov** [reply]
> There is a `postStartCommand` that will run for all container starts (i.e on create and resume) - will that work for your scenario or do you need something that only runs on resume? /cc @lostintangent

Is that currently supported by Codespaces? I see it in https://code.visualstudio.com/docs/remote/devcontainerjson-reference but it's not listed in https://docs.microsoft.com/en-us/visualstudio/codespaces/reference/configuring#codespaces-configuration-reference

**Reply by 2percentsilk** [reply]
Hi @mchelen-gov, that second doc you linked is unfortunately out of date as the Visual Studio Codespaces service has been retired. However, looks like the first doc also needs an update since `postStartCommand` is now supported for Codespaces :)

**Reply by acrollet** [reply]
Confirmed that this [works for our use case](https://github.com/department-of-veterans-affairs/va.gov-cms/commit/7e3924d2a773810b93af09eaf093078ec1af52e8) - thanks for the speedy response!

**Reply by mchelen-gov** [reply]
@2percentsilk @acrollet oh that's awesome, ty for the info! opened [PR](https://github.com/microsoft/vscode-docs/pull/4394) to update docs. any idea if other config settings are now supported as well?
